samples/reactjs-todo/client/components/journey/form.js

```diff
@@ -121,6 +125,10 @@ export default function Form({ action, bottomMessage, followUp, topMessage }) {
         return <TermsConditions callback={cb} inputName={name} key={name} />;
       case 'KbaCreateCallback':
         return <Kba callback={cb} inputName={name} key={name} />;
+      case 'TextOutputCallback':
+        return <TextOutput key={`textOutput-${idx}`} />; //For textOutput call backs, 'input' field comes empty which leads to a  unique-key-prop error
```

Comment:
`// For TextOutput callbacks, 'input' field comes empty which leads to a unique-key-prop error`